Sources/ArrayExtensions.swift

```diff
@@ -132,13 +132,15 @@ extension Array where Element: Equatable {
         self.remove(at: index)
     }
 
-    // EZSE: Removes all occurrences of the given object
+    /// EZSE: Removes all occurrences of the given object(s)
     public mutating func removeAll(_ elements: Element...) {
-        for element in elements {
-            for index in self.indexes(of: element).reversed() {
-                self.remove(at: index)
-            }
-        }
+        removeAll(elements)
+    }
+
+    /// EZSE: Removes all occurrences of the given object(s)
+    public mutating func removeAll(_ elements: [Element]) {
```

Comment:
Yes, you could say that (due to that limitation). We supply an additional overload to allow users to call `removeAll(_:)` with variadic as well as array arguments.